README.md

```diff
@@ -457,6 +459,16 @@ succeed. If your users are coming in over the Internet (and not over LAN), this
 The public IP address is discovered via [STUN]. STUN servers can be specified with the ``JVB_STUN_SERVERS``
 option.
 
+### Prosody WebSocket Support
+
+By default communication with Prosody is done using HTTP.
+
+If `ENABLE_XMPP_WEBSOCKET` is set to `1` the preferred communication method is WebSocket. HTTP is still available as a fallback.
+
+If Jitsi runs behind a reverse proxy make sure the reverse proxy configuration supports WebSocket.
+
+Read more: http://nginx.org/en/docs/http/websocket.html
+
```

Comment:
These changes need to be applied to https://github.com/jitsi/handbook/blob/master/docs/devops-guide/docker.md (cf: #601)